config/locales/es.yml

```
@@ -1691,6 +1691,7 @@
     execution_reports: "Reportes de ejecución"
     findings: "Hallazgos"
     groups: "Grupos"
+    licenses: 'Licencia'
```

```suggestion
    licenses: "Licencia"
```

Solo por consistencia con los _hermanos_.

---

Quedó _resuelto_, pero sigue con `'`, lo menciono por si se faltó un commit.

---

Disculpá :+1: